plugins/modules/mssql_script.py

```diff
@@ -56,6 +56,12 @@
           - Each batch must return at least one result set.
         required: true
         type: str
+    transaction:
+        description:
+          - If transactional mode is requested, start a transaction and commit the change only if the script succeed else rollback the transaction.
```

Comment:
```suggestion
          - If transactional mode is requested, start a transaction and commit the change only if the script succeed.
            Otherwise, rollback the transaction.
```